app/src/main/java/com/framgia/englishconversation/utils/audio/AudioPlayerRecordingImp.java

```diff
@@ -0,0 +1,88 @@
+package com.framgia.englishconversation.utils.audio;
+
+import android.app.Activity;
+import android.media.MediaPlayer;
+
+import com.framgia.englishconversation.R;
+import com.framgia.englishconversation.utils.navigator.Navigator;
+
+import java.io.IOException;
+
+/**
+ * Created by fs-sournary.
+ * Date on 12/19/17.
+ * Description:
+ */
+
+public class AudioPlayerRecordingImp implements AudioPlayerManager {
+
+    private static final int DEFAULT_AUDIO_TIME = 0;
```

Comment:
có thể tái sử dụng lại thằng ZERO_PERCENT trong Constant

Reply:
ZERO_PERCENT is only used for 0% if we use for DEFAULT_AUDIO_TIME for this class that may be occurred the mistake

Reply:
Ok sorry

Reply:
nhưng đó là hằng số và nó không thay đổi mà. a thấy nếu trường hợp e sử dụng trong một function nào đó mà chỉ cần lấy thông số đó thôi.

Reply:
single repository priciple dù là dùng cho class nhưng cũng nên làm cho nó để có thể rõ ràng đối với các member khác nếu chúng ta thay đổi thì chúng ta có thể làm được anh ạ